include/wx/stattext.h

```diff
@@ -72,6 +77,12 @@ class WXDLLIMPEXP_CORE wxStaticTextBase : public wxControl
     // display.
     void UpdateLabel();
 
+    // save unwrapped label to allow to call Wrap() several times and 
+    // always starting from the original, unwrapped label
+    wxString m_unwrappedLabel;
+    int m_currentWrap = 0;
```

Comment:
What is the value of this? Wrap width? With `0` meaning "none"? Please comment it too.